### DISCUSSION TITLE
Paid hobby plan?

### DISCUSSION THREAD

**Discussion by nooblyf** [planetscale/discussion#582] [Q&A] [answered]
Is there a way to create more than 1 database under hobby plan even if it is paid one? Scaler plan is too much for me both price and resource wise

**Comment by orware** [comment, marked as answer]
Hello @nooblyf,

Unfortunately, there isn't a way to do so at this time.

We primarily only allow for one free Hobby plan database because of the amount of infrastructure that gets provisioned for the mini Vitess + MySQL cluster you are provided with for every database created in our environment. Providing that amount of infrastructure is fairly resource intensive and doesn't allow for many free databases to be provided inexpensively especially for users that may be more cost conscious.

For other providers that are provisioning more vanilla MySQL 8 instances for you to use things work differently since within that same amount of hardware you might be paying for you can typically create many small databases without any additional costs since all of the databases are sharing that same hardware within the MySQL 8 instance.

In our environment, each of those databases is being provided its own cluster's worth of resources so the cost dynamics are considerably different, but once things are framed in that way the reason for our pricing tends to become more understandable when I've explained this to other users via ticket in our support system so I hope this extra information is helpful in understanding why creating extra Hobby plan databases isn't currently available as an option.

**Reply by nooblyf** [reply]
Understandable, thanks for the detailed explanation!

**Reply by nooblyf** [reply]
If there will be any future consideration on providing a paid hobby plan, just putting it out there that i'll be glad to pay something like $15/month for it, love using planetscale.

**Comment by abdulazizahwan** [comment]
Which countries is the PlanetScale hobby tier available in?
